config/routes.rb

```diff
@@ -7,10 +7,9 @@
   resources :orders
   resources :categories
 
-  namespace 'admin' do 
+  namespace 'admin' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.